### PR TITLE
Allow package.json to explicitly set em:multiprocessCompatible to false

### LINKED ISSUE #607: Set multiprocessCompatible flag for package.json multiprocess permission regardless of value
Per [The Road to Firefox 57](https://blog.mozilla.org/addons/2017/02/16/the-road-to-firefox-57-compatibility-milestones/), extensions that are not e10s compatible must explicitly set the multiprocessCompatible flag to false in install.rdf. For some add-ons such as 1Password, the add-on SDK is never going to be fully e10s compatible for all the APIs that are in use so setting this flag to an explicitly false value will be required.

Setting package.json's multiprocess permission to true causes the `<em:multiprocessCompatible>true</em:multiprocessCompatible>` tag to be added, but setting it to false, causes the tag to be omitted. This is clear from the code at [rdf.js:105](https://github.com/mozilla-jetpack/jpm/blob/4561d481885f732bf30fcc77044583fe6fe028fd/lib/rdf.js#L105). Without an else branch here, the extension must create its own complete install.rdf _and_ ensure it is updated in the xpi during build. Instead, jpm should respect the presence and value of the multiprocess permission in package.json regardless of its value.

### PR DESCRIPTION
**Pull request by jxpx777**
Fixes #607  - All unit tests (`npm run unit`) pass: `125 passing (1s)`

### CODE REVIEW

File: test/unit/test.rdf.js [hunk 1]
```diff
@@ -203,8 +203,8 @@ describe("lib/rdf", function() {
       expect(getData(xml, "em:multiprocessCompatible")).to.be.equal("true");
 
       xml = parseRDF(RDF.createRDF({ id: "1", permissions: { multiprocess: false } }));
-      expect(nodeExists(xml, "em:multiprocessCompatible")).to.be.equal(false);
-      expect(getData(xml, "em:multiprocessCompatible")).to.be.equal(undefined);
+      expect(nodeExists(xml, "em:multiprocessCompatible")).to.be.equal(true);
+      expect(getData(xml, "em:multiprocessCompatible")).to.be.equal("false");
```
Comment (kumar303) [inline test/unit/test.rdf.js]: How convenient 💰 I didn't know there was a test for it already

**Comment by caitmuenster** [comment]
Hi @jxpx777! Thanks for submitting the patch. Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition#March_2017). 

I'd love to help you create a profile on [mozillians.org](https://mozillians.org/en-US/). If you could ping me when you've set up your profile, I can vouch for your contributions. 

Thanks again, and welcome onboard!

**Comment by jxpx777** [comment]
Right on, @caitmuenster. Thanks! https://mozillians.org/en-US/u/jxpx777/

**Comment by caitmuenster** [comment]
Woot! Vouch completed.  🎉

Looking forward to seeing you around the web!

**Pull request merged**